ts/features/itwallet/common/components/ItwUpcomingWalletBanner.tsx

```diff
@@ -17,16 +17,22 @@ export const ItwUpcomingWalletBanner = () => {
   }
 
   return (
-    <Banner
-      color="neutral"
-      size="big"
-      pictogramName="notification"
-      title={I18n.t("features.itWallet.discovery.upcomingWalletBanner.title")}
-      content={I18n.t(
-        "features.itWallet.discovery.upcomingWalletBanner.content"
-      )}
-      action={I18n.t("features.itWallet.discovery.upcomingWalletBanner.action")}
-      onPress={() => Linking.openURL("https://io.italia.it/")} // TODO: [SIW-1716] Update the URL
-    />
+    <>
+      <Banner
+        testID="itwUpcomingWalletBannerTestID"
+        color="neutral"
+        size="big"
+        pictogramName="notification"
+        title={I18n.t("features.itWallet.discovery.upcomingWalletBanner.title")}
+        content={I18n.t(
+          "features.itWallet.discovery.upcomingWalletBanner.content"
+        )}
+        action={I18n.t(
+          "features.itWallet.discovery.upcomingWalletBanner.action"
+        )}
+        onPress={() => Linking.openURL("https://io.italia.it/documenti-su-io/")}
+      />
+      <VSpacer size={24} />
+    </>
```

Comment:
Same goes for this `VSpacer`, should we use it outside this component? Does it make sense to you?

Reply:
Yes, you're right. I'll include an optional prop for the vertical space